src/Talk/Create/CreateTalkFormType.php

```diff
@@ -13,7 +12,7 @@
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-final class TalkType extends AbstractType
+class CreateTalkFormType extends AbstractType
```

Comment:
Why removing the final? If you want to go down that road, I would rather have an abstract TalkFormType and two final Create/Update, that would make much more sense to me (I can't see edit inheriting from create really on a purely functional standpoint).
I personally would rather be in favour of duplication instead of premature/useless factorisation.

Anyhow, I would leave it as it was for now, and probably go for another issue/PR on this specific matter to keep the discussion clean and opened.